src/plugins/node-workspace.ts

```diff
@@ -297,16 +315,17 @@ export class NodeWorkspace extends WorkspacePlugin<Package> {
           createIfMissing: false,
           updater: new Changelog({
             version,
-            changelogEntry: appendDependenciesSectionToChangelog(
-              '',
-              dependencyNotes,
-              this.logger
-            ),
+            versionHeaderRegex: `\n###? v?[${version
```

Comment:
The base PR will include the version. The default regex would insert changes above the last version. Which makes it backward.

I need to come back and cleanup the "append" updater if the PR moves forward.